go/store/datas/commit.go

```diff
@@ -170,9 +170,9 @@ func parentsToQueue(ctx context.Context, refs types.RefSlice, q *types.RefByHeig
 			}
 			if ok {
 				p := ps.(types.Set)
-				err = p.IterAll(ctx, func(v types.Value) error {
+				err = p.Iter(ctx, func(v types.Value) (stop bool, err error) {
 					q.PushBack(v.(types.Ref))
-					return nil
+					return
 				})
 			}
```

Comment:
these changes look functionally equivalent.  Just a style change or is there something I'm missing here?

Reply:
it's using `Iter` instead of `IterAll`